src/nCr/index.js

```diff
@@ -0,0 +1,36 @@
+/* nCr
+ * Function: nCr()
+ */
+
+const validate = require('../validation/non-negative-integer');
+
+/**
+ *
+ * @param {number, number} number
+ * @returns {number}
+ */
+
+function nCr(number1, number2) {
+
+	if(number1 < number2){
```

Comment:
This piece of code shouldn't be here, as it is already included in the validation tests and further checked when exporting the module.

Reply:
But there is no file in the validation folder, that compares two numbers.